database_functions.py

```diff
@@ -7,7 +7,7 @@ def get_entries(user_id):
 
 
 def deleteEntry(entry_id):
-    entry = Entry.query.filter_by(id=entry_id)
+    entry = Entry.query.filter_by(id=entry_id).first()
```

Comment:
I forgot to include ".first()" originally, but with this quick edit, I can pull the entry that we need to be deleted.